tests/integration/iop/test_dag_process_file.py

```diff
@@ -0,0 +1,149 @@
+from zipfile import ZipFile
+
+import pytest
+from airflow import DAG
+from airflow.models import DagBag
+from common.utils import parse_to_ET_element
+from freezegun import freeze_time
+from iop.iop_process_file import iop_enhance_file, iop_enrich_file, iop_validate_record
+from iop.parser import IOPParser
+from pytest import fixture
+
+DAG_NAME = "iop_process_file"
+
+
+@fixture
+def dag():
+    dagbag = DagBag(dag_folder="dags/", include_examples=False)
+    assert dagbag.import_errors.get(f"dags/{DAG_NAME}.py") is None
+    return dagbag.get_dag(dag_id=DAG_NAME)
+
+
+@fixture
+def parser():
+    return IOPParser()
+
+
+@fixture
+def article(parser):
+    data_dir = "./data/iop/"
+    test_file = "2022-07-30T03_02_01_content.zip"
+
+    def extract_zip_to_article(zip_filename):
+        with ZipFile(zip_filename, "r") as zip_file:
+            return [
+                zip_file.read(filename)
+                for filename in map(lambda x: x.filename, zip_file.filelist)
```

Comment:
map? :((( it's unpythonic, see the discussion [https://stackoverflow.com/questions/1247486/list-comprehension-vs-map](there) 😝 

But jokes aside, in this example it's reasonable to use a list comprehension cause you don't need lambda to perform any kind of operation on the list. You can also split this return in 2 statements for readability.